components/Hero/styles.css

```diff
@@ -45,6 +45,20 @@
   z-index: 1;
 }
 
+.Hero-links {
+  position: relative;
+  max-width: 1024px;
+  height: 40px;
+  margin-top: -45px;
+}
```

Comment:
out of curiousity, this is only for the home and hamburger, right? Are these supposed to be constrained, too? cc @shieldsma